build.gradle

```diff
@@ -10,7 +10,7 @@ ext.hadoopBinaries = "${rootDir}/hadoop-binaries".toString()
 ext.javaDriverVersion = '3.2.1'
 ext.hiveVersion = '1.2.1'
 ext.pigVersion = '0.15.0'
-ext.hadoopVersion = '2.7.2'
+ext.hadoopVersion = '1.2.1'
```

Comment:
I'm not gradle  expert, but  I don't think so. You can move this property to external file, but I've never seen ways to specify it in command line. Also  v1 and v2 have  different libraries, so having version dynamic won't help